source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -72,221 +84,20 @@ Third party libraries are packaged into Python wheels and uploaded to PyPI (if p
 
       The ``vendordep`` gradle task can also fetch vendordep JSONs from the user ``wpilib`` folder. To do so, pass ``FRCLOCAL/Filename.json`` as the file URL. For example, ``gradlew vendordep --url=FRCLOCAL/WPILibNewCommands.json`` will fetch the JSON for the command-based framework.
```

Comment:
This Command-Line section feels like it should be another tab, rather then hidden in the legacy VS Code section